docs/_data/toc.yml

```diff
@@ -141,7 +141,7 @@
 - title: 📖 Reference
   url: reference
   links:
-    - title: mesheryctl Commands
+    - title: Mesheryctl Commands
```

Comment:
Please revert, @warunicorn19. The proper name of the binary is `mesheryctl` -lowercase. The generic reference is "Meshery CLI" - capital.

Reply:
For sake of uniformity with other page titles that are capitalized, this page title can be changed to "Meshery CLI", if you like. I believe it was this name in the past. There has been some flip-flopping on which way to go.

Reply:
Let's switch to Meshery CLI then, it's a minor detail but looks a little weird when you see it since everything is capitalized.

Reply:
My OCD agrees with you.